browser-extension/manifest.json

```diff
@@ -34,6 +35,6 @@
     }
   ],
   "web_accessible_resources": [
-	  "rate_now_icon.png"
+	  "Logo128.png"
```

Comment:
This filename also needs to be updated in addRateLaterButton.js : 
https://github.com/tournesol-app/tournesol/blob/3a1ff590e693dad3035f9b4a7ca395a830efac1f/browser-extension/addRateLaterButton.js#L56